python/python/glide/async_commands/cluster_commands.py

```diff
@@ -367,3 +368,87 @@ async def lastsave(self, route: Optional[Route] = None) -> TClusterResponse[int]
             TClusterResponse[int],
             await self._execute_command(RequestType.LastSave, [], route),
         )
+
+    async def sort(
+        self,
+        key: str,
+        limit: Optional[Limit] = None,
+        order: Optional[OrderBy] = None,
+        alpha: Optional[bool] = None,
+    ) -> List[str]:
+        """
+        Sorts the elements in the list, set, or sorted set at `key` and returns the result.
```

Comment:
We should duplicate the absolute minimum from the official docs and add nothing that is not in official docs. This makes it easier adhering to a single source of truth (docs) which in turn helps with maintenance (changes in valkey docs do not require changing our docs ) and minimizes confusion (customers refer to the single source of information)
In short the doc should be:
```
Returns or stores the elements contained in the list, set or sorted set at key.
See https://valkey.io/commands/sort for more details.
```

Reply:
@shohamazon WDYT?